pyo3-ffi/src/cpython/lock.rs

```diff
@@ -1,11 +1,23 @@
-use std::marker::PhantomPinned;
 use std::sync::atomic::AtomicU8;
 
 #[repr(transparent)]
 #[derive(Debug)]
 pub struct PyMutex {
     pub(crate) _bits: AtomicU8,
-    pub(crate) _pin: PhantomPinned,
```

Comment:
What was the motivation to remove the `_pin`? I have a feeling we agreed this was necessary previously.

Reply:
Oops! This is leftover from last year. I hope me deleting that wasn't load-bearing. Thanks very much for catching this.